### PR TITLE
Minor tweaks to changelog and upgrade.

### PR DESCRIPTION
**Pull request by eschwab**
- Also found two deprecated macros to convert to the new deprecated
  system.

- Revert `GreDagger` to `gredagger` since it does not appear in `gregoriotex-write.c`

### CODE REVIEW

**Comment by henryso** [comment]
GreDagger *does* appear in gregoriotex-write.c, twice.

**Comment by eschwab** [comment]
Yes it does. Looks like I was grepping `gabc-write` instead. Since `GreDagger` can be both a gregorio and user level macro, would it be useful  to define both `GreDagger` and `gredagger` similar to how we have `GreStar` and `gresixstar`?

**Comment by henryso** [comment]
The difference is that `\GreStar` is an alias for `\gresixstar`, which has an alternate, `\greheightstar`.  So you would issue `\let\GreStar\greheightstar` to select the alternate, whereas the same is not true of `\GreDagger`, which is the "one-and-only" dagger figure.  Of course the waters are very muddy here because of the new alternates system.

It really doesn't matter to me one way or the other, so if people are in favor of defining `\gredagger` and aliasing `\GreDagger` to it, I don't have any objection.

File: CHANGELOG.md [hunk 1]
```diff
@@ -32,7 +42,7 @@ As of v3.0.0 this project adheres to [Semantic Versioning](http://semver.org/).
...
   - It is now possible to suppress the automatic horizontal episema "bridging" across spaces within a syllable.  To use it, append `2` to the `_` on the note before the space you do not want bridged (see [#72](https://github.com/gregorio-project/gregorio/issues/72)).
   - A horizontal episema will no longer overlap with a vertical episema on the same note (see [#395](https://github.com/gregorio-project/gregorio/issues/395)).
-- The dot after a torculus whose last note is on a line will now appear below the line.  If you do not like the automatic placement of a dot after any note on a line (not just the torculus), you may specify a `0` or `1` after the `.` to force the dot to appear below or above the line, respectively.  See [#386](https://github.com/gregorio-project/gregorio/issues/386).
+- The dot after a torculus whose last note is on a line will now appear below the line (see [#386](https://github.com/gregorio-project/gregorio/issues/386).)  If you do not like the automatic placement of a dot after any note on a line (not just the torculus), you may specify a `0` or `1` after the `.` to force the dot to appear below or above the line, respectively.
```
Comment (henryso) [inline CHANGELOG.md]: The period should come after the right parenthesis.

File: tex/gregoriotex.sty [hunk 2]
```diff
@@ -125,12 +125,12 @@
 
 
 \def\grecoloredlines#1#2#3{%
-  \gre@warning{\protect\grecoloredlines\space is deprecated.\MessageBreak Use \protect\gresetlinecolor\space instead.}%
+  \gre@deprecated{\protect\grecoloredlines}{\protect\gresetlinecolor{red}}%
```
Comment (henryso) [inline tex/gregoriotex.sty]: I'm not sure if "red" should be mentioned here.  The command is not exactly equivalent in any case, so perhaps just leave the argument out, like the original warning?  I can't think of a good, terse way to do this here.

**Comment by henryso** [comment]
Besides the last comment, I'm fine with this pull request being merged.

File: tex/gregoriotex.sty [hunk 3]
```diff
@@ -123,14 +123,13 @@
...
-
 \def\grecoloredlines#1#2#3{%
-  \gre@warning{\protect\grecoloredlines\space is deprecated.\MessageBreak Use \protect\gresetlinecolor\space instead.}%
+  \gre@deprecated{\protect\grecoloredlines}{\protect\gresetlinecolor}%
   \gre@changestyle{normalstafflines}{\color[RGB]{#1,#2,#3}}%
 }
 
 \def\greredlines{%
-  \gre@warning{\protect\greredlines\space is deprecated.\MessageBreak Use \protect\gresetlinecolor{gregoriocolor}\space instead.}%
+  \gre@deprecated{\protect\greredlines}{\protect\gresetlinecolor{gregoriocolor}%
```
Comment (henryso) [inline tex/gregoriotex.sty]: Sorry for being piecemeal, but I didn't notice it before.  The braces are unbalanced in #2.  Same below.

**Comment by henryso** [comment]
Looks good.

**Pull request merged**